Tasks/UsePythonVersion/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "use-python-version",
+  "version": "0.1.0",
+  "description": "Retrieves the specified version of Python from the tool cache. Optionally add it to PATH.",
+  "main": "usepythonversion.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/vsts-tasks.git"
+  },
+  "keywords": [
+    "VSTS",
+    "Tasks",
+    "Python"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "@types/mocha": "^2.2.48",
```

Comment:
All these except task-lib and tool-lib are dev dependencies.  I wanted to install them as such, which would keep them from being bundled with the task for on-prem distribution and make registering the OSS simpler.  However, the build system currently throws an error if the package.json contains dev dependencies.